examples/mariadb-centos7-atomicapp/artifacts/kubernetes/mariadb-pod.yaml

```diff
@@ -4,7 +4,7 @@ spec:
   containers:
     - resources:
```

Comment:
I suggest we remove the resources section altogether and let the the manifest be more generic to suit different cluster sizes.